internal/credential/vault/testing.go

```diff
@@ -0,0 +1,47 @@
+package vault
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/boundary/internal/db"
+	wrapping "github.com/hashicorp/go-kms-wrapping"
+	"github.com/jinzhu/gorm"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+// TestCredentialStores creates count number of vault credential stores in
+// the provided DB with the provided scope id. If any errors are
+// encountered during the creation of the credential stores, the test will
+// fail.
+func TestCredentialStores(t *testing.T, conn *gorm.DB, wrapper wrapping.Wrapper, scopeId string, count int) []*CredentialStore {
```

Comment:
Any reason to export this test helper

Reply:
ty!

Reply:
Yes, I expect it will be needed in other packages since it is an aggregate root. This is [fairly common](https://github.com/hashicorp/boundary/blob/93bad3cb83626d87fce5ed4dad0e5e502eb71700/internal/session/testing.go#L104) in the code base.